src/extensions/resolveSource.ts

```diff
@@ -9,12 +10,16 @@ import { promptOnce } from "../prompt";
 
 const EXTENSIONS_REGISTRY_ENDPOINT = "/extensions.json";
 const AUDIENCE_WARNING_MESSAGES: { [key: string]: string } = {
-  "open-alpha": `${clc.bold(
-    "Important"
-  )}: This extension is part of the preliminary-release program for extensions. Its functionality might change in backward-incompatible ways before its official release.`,
-  "closed-alpha": `${clc.yellow.bold(
-    "Important"
-  )}: This extension is part of the Firebase Alpha program. This extension is strictly confidential, and its functionality might change in backward-incompatible ways before its official, public release.`,
+  "open-alpha": marked(
+    `${clc.bold("Important")}: This extension is part of the ${clc.bold(
+      "preliminary-release program"
+    )} for extensions. Its functionality might change in backward-incompatible ways before its official release. Learn more: https://github.com/firebase/extensions/tree/next/.preliminary-release-extensions`
+  ),
+  "closed-alpha": marked(
+    `${clc.yellow.bold("Important")}: This extension is part of the ${clc.bold(
+      "Firebase Alpha program"
+    )}. This extension is strictly confidential, and its functionality might change in backward-incompatible ways before its official, public release. Learn more: https://dev-partners.googlesource.com/samples/firebase/extensions-alpha/`
```

Comment:
This is a better link: https://dev-partners.googlesource.com/samples/firebase/extensions-alpha/+/refs/heads/master/README.md